apps/api/src/lib/db/migrations/0039_complex_hawkeye.sql

```
@@ -0,0 +1,10 @@
+CREATE TABLE "discord_challenge_map_history" (
+	"id" bigserial PRIMARY KEY NOT NULL,
+	"batch_id" text NOT NULL,
+	"map_id" bigint NOT NULL,
+	"selected_at" integer NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "discord_challenge_map_history" ADD CONSTRAINT "discord_challenge_map_history_map_id_maps_id_fk" FOREIGN KEY ("map_id") REFERENCES "public"."maps"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)